.github/workflows/publish-release.yml

```diff
@@ -62,12 +62,19 @@ jobs:
           git push --tags
 
       # Create Draft Release on GitHub with tag of version
-      - name: Create Draft Release
+      - name: Create Draft Release (`apollo-ios`)
         shell: bash
         run: |
           gh release create ${{ env.RELEASE_VERSION }} 'apollo-ios/CLI/apollo-ios-cli.tar.gz#apollo-ios-cli.tar.gz' -d --repo "apollographql/apollo-ios" -t "${{ env.RELEASE_VERSION }}"
         env:
           GH_TOKEN: ${{ secrets.APOLLO_IOS_PAT }}
+      
+      - name: Create Draft Release (`apollo-ios-codegen`)
+        shell: bash
+        run: |
+          gh release create ${{ env.RELEASE_VERSION }} -d --repo "apollographql/apollo-ios-codegen" -t "${{ env.RELEASE_VERSION }}"
+        env:
+          GH_TOKEN: ${{ secrets.APOLLO_IOS_PAT }}
```

Comment:
@BobaFetters - do you know if this PAT will allow us to create a release in the `apollo-ios-codegen` repo too? I noticed that we're using it in the step below for triggering the `apollo-ios-xcframework` release workflow so I'm hopeful the answer is yes?

Reply:
sorry not sure how I didn't see this PR sooner, yea the PAT works for all the apollo ios repos so we can use the same one